src/components/subscriptions/addons/edit/EditAddon.js

```diff
@@ -239,6 +245,35 @@ function EditAddonForm(props) {
                 label={t("defaultPaid")}
                 name="defaultPaid"
             />
+            <FieldArray
+                name="addonRates"
+                render={(arrayHelpers) => {
+                    const onAdd = () => {
+                        arrayHelpers.push({
+                            uuid: crypto.randomUUID,
```

Comment:
This should be a function call:
```suggestion
                            uuid: crypto.randomUUID(),
```

Reply:
That's embarrassing. I tested it, and it worked which is a little puzzling. I'll commit your suggestion.

Reply:
Looks like it was assigning a reference to the function, which is valid, but if you added more than one addon rate,  then the browser console should have shown a warning about duplicate key values (and I imagine the API submission would fail too).